server/src/test/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryPluginTest.java

```diff
@@ -3,69 +3,107 @@
-import static org.opentosca.toscana.plugins.cloudfoundry.FileCreator.FILEPRAEFIX_DEPLOY;
-import static org.opentosca.toscana.plugins.cloudfoundry.FileCreator.FILESUFFIX_DEPLOY;
-import static org.opentosca.toscana.plugins.cloudfoundry.FileCreator.MANIFEST_PATH;
-import static org.opentosca.toscana.plugins.cloudfoundry.FileCreator.deploy_name;
-import static org.opentosca.toscana.plugins.scripts.BashScript.SHEBANG;
-import static org.opentosca.toscana.plugins.scripts.BashScript.SOURCE_UTIL_ALL;
-import static org.opentosca.toscana.plugins.scripts.BashScript.SUBCOMMAND_EXIT;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.opentosca.toscana.plugins.cloudfoundry.CloudFoundryPlugin.CF_PROPERTY_KEY_API;
+import static org.opentosca.toscana.plugins.cloudfoundry.CloudFoundryPlugin.CF_PROPERTY_KEY_ORGANIZATION;
+import static org.opentosca.toscana.plugins.cloudfoundry.CloudFoundryPlugin.CF_PROPERTY_KEY_PASSWORD;
+import static org.opentosca.toscana.plugins.cloudfoundry.CloudFoundryPlugin.CF_PROPERTY_KEY_SPACE;
+import static org.opentosca.toscana.plugins.cloudfoundry.CloudFoundryPlugin.CF_PROPERTY_KEY_USERNAME;
+import static org.opentosca.toscana.plugins.cloudfoundry.filecreator.FileCreator.FILEPRAEFIX_DEPLOY;
+import static org.opentosca.toscana.plugins.cloudfoundry.filecreator.FileCreator.FILESUFFIX_DEPLOY;
+import static org.opentosca.toscana.plugins.cloudfoundry.filecreator.FileCreator.MANIFEST_PATH;
+import static org.opentosca.toscana.plugins.cloudfoundry.filecreator.FileCreator.deploy_name;
 import static org.opentosca.toscana.plugins.util.TestUtil.setUpMockTransformationContext;
 
 public class CloudFoundryPluginTest extends BaseUnitTest {
 
-    private File targetDir;
-    private final String appName = "my_app";
     private final ArrayList<String> paths = new ArrayList<>();
     private final String resourcesPath = "src/test/resources/";
 
+    @Mock
     private TransformationContext context;
+    private EffectiveModel lamp;
+    private File targetDir;
+
+    private Graph<RootNode, RootRelationship> topology;
+    private CloudFoundryLifecycle cfCycle;
 
     @Before
     public void setUp() throws Exception {
-        EffectiveModel lamp = new EffectiveModelFactory().create(TestCsars.VALID_LAMP_NO_INPUT_TEMPLATE, logMock());
-        this.context = setUpMockTransformationContext(lamp);
-        Application myApp = new Application(appName, 1, context);
-        NodeVisitor visitor = new NodeVisitor(myApp);
+        lamp = new EffectiveModelFactory().create(TestCsars.VALID_LAMP_NO_INPUT_TEMPLATE, logMock());
+        context = setUpMockTransformationContext(lamp);
+
+        String userName = System.getenv("TEST_CF_USER");
+        String pw = System.getenv("TEST_CF_PW");
+        String host = System.getenv("TEST_CF_HOST");
+        String space = System.getenv("TEST_CF_SPACE");
+        String orga = System.getenv("TEST_CF_ORGA");
+        Assume.assumeNotNull(userName);
```

Comment:
you can also combin all assumtions into one `assumeNotNull(userName, pw, host...)`